index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+  
+    <header class="header">
+      <h3 class="head">Welcome</h3>
```

Comment:
- [ ] Your toolbar is great, but it is usual that the logo of a page is a link to the home page, so an `a` tag would be better for the logo. Kindly change the tag for better user experience